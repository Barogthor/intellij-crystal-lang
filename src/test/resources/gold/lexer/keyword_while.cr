while some_condition
  do_this
end