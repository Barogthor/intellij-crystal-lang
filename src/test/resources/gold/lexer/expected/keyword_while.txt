identifier ('while')
whitespace (' ')
identifier ('some_condition')
whitespace ('\n  ')
identifier ('do_this')
whitespace ('\n')
end ('end')